docs/team/member-guide.md

```diff
@@ -0,0 +1,105 @@
+(membership_guidelines)=
```

Comment:
There are a lot of mentions of "team" in this file that should be changed to "council". Also, it would be great to change the directory of these files from "team" to "council" if these are all about the council.

Reply:
I think there are a few mentions of "team" that should mean the wider community of people. And there are some places where "team member" should just be "member". But I do think we should retain *some* usage of the term "team" because this is, after all, the `team-compass`.

Reply:
> But I do think we should retain _some_ usage of the term "team" because this is, after all, the `team-compass`.

Yes, absolutely!